apps/publisher/themes/default/js/create_asset.js

```diff
@@ -37,9 +37,14 @@ $(function () {
             }
         });
         //Populate the selected tags
-        tagEntry.value = tagsAPI.selectedTags();
-        modifiedArr.push(tagEntry);
-        arr = modifiedArr;
+        if(tagEntry == undefined) {
```

Comment:
Done the newly requested change https://github.com/wso2/carbon-store/pull/843/files#diff-bafe24e9fdc100f9f89a823370eb6b3d